### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [cognitive-engineering-lab/argus#58]
Technically, Argus is (mostly) accessible, since it operates by expanding various aspects of the UI on click (which mostly seem to have `aria-expanded` attributes to signal that they can get expanded).

However, it heavily relies on icons and (probably?) coloring to convey critical information and that unfortunately doesn't work for screen readers. It's possible this is easy to fix with appropriate `aria-labels` and `sr-only` text, but I'm not sure.

The other problem occurs where hovering with a mouse to display full paths is required: This isn't just a touch screen problem but is also potentially inaccessible for screen reader users (unless they know how to control the mouse pointer using their screen reader) and people with physical impairments which make hovering with a mouse difficult.

(Or simply those who prefer to use a keyboard only with no mouse.)
